deployment/deployutils/deployutils.cpp

```diff
@@ -827,6 +827,12 @@ class CGenerateJSFromXSD
               LoadComboBox("Software/DaliServerProcess", bAddBlank, m_pEnv, m_pEnv, strBuf);
               extraInfo = strBuf.str();
             }
+            else if (strcmp(type, "addrCleanerPluginType")==0)
```

Comment:
This does reference the optional plugin by name but didn't see a way around that.